src/locales/srs-ui.json

```diff
@@ -0,0 +1,133 @@
+{
+  "common": {
+    "loading": "Loading",
```

Comment:
I think this service registry translation file should be in ui-components repo so that we can reuse for SR  storybook  components.